From b22ec1dae520bdbe836921acb005120ac02f545b Mon Sep 17 00:00:00 2001
From: Nirbheek Chauhan <nirbheek@centricular.com>
Date: Thu, 14 Mar 2019 18:24:57 +0530
Subject: [PATCH] meson: Disable tests, fail on our ancient MinGW

FAILED: glib/tests/a20e7a2@@date@exe/date.c.obj
x86_64-w64-mingw32-gcc @glib/tests/a20e7a2@@date@exe/date.c.obj.rsp
../glib/tests/date.c: In function 'test_month_names':
../glib/tests/date.c:332:20: error: 'SUBLANG_LITHUANIAN_LITHUANIA' undeclared (first use in this function)
../glib/tests/date.c:332:20: note: each undeclared identifier is reported only once for each function it appears in
---
 glib/meson.build | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/glib/meson.build b/glib/meson.build
index ea9bf41..4b8a48d 100644
--- a/glib/meson.build
+++ b/glib/meson.build
@@ -328,6 +328,6 @@ if enable_systemtap
 endif
 
 # gtester doesn't work on native windows
-if cc.get_id() != 'msvc'
-  subdir('tests')
-endif
+#if cc.get_id() != 'msvc'
+#  subdir('tests')
+#endif
-- 
2.20.1.windows.1

